File: One/Employee.php
<?php

class Employee {
    private string $name;
    private string $identification;
    private float $baseSalary;
    private string $arlRiskLevel;

    public function __construct(string $name, string $identification, float $baseSalary, string $arlRiskLevel) {
        $this->name = $name;
        $this->identification = $identification;
        $this->baseSalary = $baseSalary;
        $this->arlRiskLevel = $arlRiskLevel;
    }

    public function getName(): string {
        return $this->name;
    }

    public function getIdentification(): string {
        return $this->identification;
    }

    public function getBaseSalary(): float {
        return $this->baseSalary;
    }

    public function getArlRiskLevel(): string {
        return $this->arlRiskLevel;
    }

    public function setName(string $name): void {
        $this->name = $name;
    }

    public function setIdentification(string $identification): void {
        $this->identification = $identification;
    }

    public function setBaseSalary(float $baseSalary): void {
        $this->baseSalary = $baseSalary;
    }

    public function setArlRiskLevel(string $arlRiskLevel): void {
        $this->arlRiskLevel = $arlRiskLevel;
    }
}

?>

File: java/One/One/Employee.java
public class Employee {
    private String name;
    private String identification;
    private double baseSalary;
    private String arlRiskLevel;

    public Employee(String name, String identification, double baseSalary, String arlRiskLevel) {
        this.name = name;
        this.identification = identification;
        this.baseSalary = baseSalary;
        this.arlRiskLevel = arlRiskLevel;
    }

    public String getName() {
        return name;
    }

    public String getIdentification() {
        return identification;
    }

    public double getBaseSalary() {
        return baseSalary;
    }

    public String getArlRiskLevel() {
        return arlRiskLevel;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setIdentification(String identification) {
        this.identification = identification;
    }

    public void setBaseSalary(double baseSalary) {
        this.baseSalary = baseSalary;
    }

    public void setArlRiskLevel(String arlRiskLevel) {
        this.arlRiskLevel = arlRiskLevel;
    }
}